### PR TITLE
update infrastructure insights tests for pipeline

### PR DESCRIPTION
**Pull request by TheOnlyWei**
Update infrastructureinsights module tests for azure devops pipeline.

Here is the successful run:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=414724&view=logs&j=6399d89d-3d30-5fff-3e92-07bbaf7ebe2f&t=fc05ba24-22a7-5af3-9197-29d15a72da23

New run:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=432131&view=results

### CODE REVIEW

File: src/Azs.InfrastructureInsights.Admin/test/Close-AzsAlert.Tests.ps1 [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
+    $envFilePath = Join-Path $PSScriptRoot $envFile
+} else {
+    $envFilePath = Join-Path $PSScriptRoot '..\$envFile'
+}
+$env = @{}
+if (Test-Path -Path $envFilePath) {
+    $env = Get-Content (Join-Path $PSScriptRoot $envFile) | ConvertFrom-Json
+    $PSDefaultParameterValues = @{"*:SubscriptionId" = $env.SubscriptionId; "*:Tenant" = $env.Tenant; "*:Location" = $env.Location; "*:ResourceGroupName" = $env.ResourceGroup }
+}
+
```
Comment (viananth) [inline src/Azs.InfrastructureInsights.Admin/test/Close-AzsAlert.Tests.ps1:17]: Please remove this section from all tests and add default parameters to utils.ps1 file as suggested here:
From Xiaogang:

If you need to add a default parameter, you should do it in the setEnv function in utils.ps1.

You may refer to https://github.com/Azure/azure-powershell/blob/generation/src/Databricks/test/utils.ps1 as an example.

If we use this method, then we won't need to add the above lines of code for every test.

File: src/Azs.InfrastructureInsights.Admin/test/common.ps1 [hunk 2]
```diff
@@ -1,4 +1,8 @@
-﻿function ValidateMetrics{
+﻿$global:SkippedTests = @("TestCloseAlert")
```
Comment (viananth) [inline src/Azs.InfrastructureInsights.Admin/test/common.ps1:1]: Why is this test skipped?

File: src/Azs.InfrastructureInsights.Admin/test/common.ps1 [hunk 3]
```diff
@@ -1,4 +1,8 @@
...
+$global:Location = "redmond"
```
Comment (viananth) [inline src/Azs.InfrastructureInsights.Admin/test/common.ps1:2]: If location & RgName are added to default parameters, its not required to hardcode here as a global variable. You can simply remove the location parameter and resource group name parameter from any cmdlet.

File: src/Azs.InfrastructureInsights.Admin/test/Close-AzsAlert.Tests.ps1 (hunk 1 above)
Comment (TheOnlyWei) [reply src/Azs.InfrastructureInsights.Admin/test/Close-AzsAlert.Tests.ps1:17]: Putting env setup in utils.ps1 didn't work for me.

File: src/Azs.InfrastructureInsights.Admin/test/common.ps1 (hunk 2 above)
Comment (TheOnlyWei) [reply src/Azs.InfrastructureInsights.Admin/test/common.ps1]: Added it back.

File: src/Azs.InfrastructureInsights.Admin/test/common.ps1 (hunk 3 above)
Comment (TheOnlyWei) [reply src/Azs.InfrastructureInsights.Admin/test/common.ps1]: Removed.

**Pull request merged**